src/parser/view_transform.c

```diff
@@ -358,6 +359,7 @@ static const char *get_authorization_name (DB_AUTH auth);
 static PT_NODE *mq_add_dummy_from_pre (PARSER_CONTEXT * parser, PT_NODE * node, void *arg, int *continue_walk);
 static PT_NODE *mq_update_order_by (PARSER_CONTEXT * parser, PT_NODE * statement, PT_NODE * query_spec,
 				    PT_NODE * class_, PT_NODE * derived_spec);
+static PT_NODE *mq_update_orderby_for (PARSER_CONTEXT * parser, PT_NODE * statement);
```

Comment:
mq_update_orderby_for() is a newly added function, but it is not used.
The comment says it needs to be removed, so why was it added to the PR?

Reply:
I'll check it. thanks.

Reply:
I removed mq_update_orderby_for ()